Classes/ConfigurationModifier.php

```diff
@@ -71,6 +71,7 @@ protected static function findPagesWithPlugins(array $withPlugins): array
             ->select('pid')
             ->from('tt_content')
             ->where(
+                $queryBuilder->expr()->in('CType', $queryBuilder->createNamedParameter('list', Connection::PARAM_STR)),
```

Comment:
Thank you for the fix!
Can you use `->eq()` here, since we're only checking for one specific value?